### ISSUE TITLE
Are you planning image upload for post thumbnails?

### ISSUE THREAD

**Issue by JohnnyEn** [sheodox/alexandrite#75]
Hi, I'm pretty excited from your Alexandrite web app, it's pretty rock solid work. Thanks for that!
Are you planning to add support for uploading image thumbnail in post composer component?

Thanks for any reply!
Regards Johnny

**Comment by sheodox** [comment]
Thanks!

Yeah, but I'm waiting for someone to do https://github.com/LemmyNet/lemmy/issues/3567, then I can add it. Otherwise I'd have to proxy all the image uploads through Alexandrite's server like Voyager and Photon do, and I really want to avoid doing that.

**Comment by JohnnyEn** [comment]
@sheodox Yup, it's workaroundable by pinning image url to URL input, but image upload could be better. Thank you for the reply!

**Comment by NaturalHate** [comment]
Is this for thumbnails in List & Compact views not showing for external URLs?

**Comment by sheodox** [comment]
Image uploading is in the new version! 
https://github.com/sheodox/alexandrite/releases/tag/0.8.16
